projects/standard-rulesets/src/examples/requireExample.ts

```diff
@@ -87,22 +87,32 @@ export const requireParameterExamples = (applies: typeof appliesWhen[number]) =>
     rule: (operation) => {
       const lifecycle = applies === 'always' ? 'requirement' : 'addedOrChanged';
       const errorMessageType = applies === 'always' ? 'every' : 'added';
+
+      function schemaHasExample(schema: OpenAPIV3.SchemaObject | OpenAPIV3.ReferenceObject | undefined) {
+        if (schema) {
+          if ('example' in schema) {
+            return typeof schema.example !== 'undefined'
+          }
+        }
+        return false
+      }
```

Comment:
Do we need to handle examples as well?

And can we also update the tests

Reply:
That's handled in the first part of the if statement 101 108 115